marbles/core/marbles/core/marbles.py

```diff
@@ -328,9 +328,19 @@ def formattedMsg(self):  # mimic unittest's name for standardMsg
             standardMsg=self.standardMsg, assert_stmt=self.assert_stmt,
             note=self.note, locals=local_string, filename=self.filename)
 
+    @classmethod
+    def _format_one_local(cls, name, value):
```

Comment:
Why `_format_one_local`? I don't understand that method name...

Reply:
It formats a single local variable. `_format_locals` formats them all, and delegates to this method. Is that clear, should I change the name, add a comment?

Reply:
Oh I get it. I think making local singular is enough? So we'd have `_format_locals` and `_format_local`. What do you think of that?